chemie/corporate/templates/corporate/interview.html

```diff
@@ -3,66 +3,169 @@
-        </p>
-        <div style="padding: 0 15px 15px 15px;">
-          {% for specialization in interview.specializations.all %}
-            <span class="badge badge-pill
+    </div>
+
+    <br><br>
+
+    <div class="row" style="margin-bottom: 80px;">
+      {% for interview in interviews %}
+        <div class="col-12 col-md-6 col-lg-4 col-xl-3" style="margin-bottom: 50px;">
+          <div class="card" style="bacground-color: var(--gray-50)">
+            <img src="{{ interview.picture.url }}" class="card-img-top" style="">
+            <p style="font-size: var(--font-size-medium-plus); font-weight: 600; padding: 15px 15px 0 15px;">
+              {{ interview.title }}
+            </p>
+            <div style="padding: 0 15px 15px 15px;">
+              {% for specialization in interview.specializations.all %}
+                <span class="badge badge-pill
              {% if specialization.id == 1 %} badge-custom-1 {% endif %}
              {% if specialization.id == 2 %} badge-custom-2 {% endif %}
              {% if specialization.id == 3 %} badge-custom-3 {% endif %}
              {% if specialization.id == 4 %} badge-custom-4 {% endif %}
              {% if specialization.id == 5 %} badge-custom-5 {% endif %}
              {% if specialization.id == 6 %} badge-custom-6 {% endif %}
               ">{{ specialization }}</span>
-          {% endfor %}
+              {% endfor %}
+            </div>
+            <a class="stretched-link" href="{% url "corporate:interview_detail" interview.id %}"> </a>
+          </div>
         </div>
-        <a class="stretched-link" href="{% url "corporate:interview_detail" interview.id %}"> </a>
-      </div>
-    </div>
 
-    {% empty %}
-    <div class="col-12" style="margin-bottom: 80px;">
-      <p class="lead">Det finnes dessverre ingen intervjuer enda...</p>
-    </div>
+      {% empty %}
+        <div class="col-12" style="margin-bottom: 80px;">
+          <p class="lead">Det finnes dessverre ingen intervjuer som passer søket ditt...</p>
+        </div>
 
-    {% endfor %}
+      {% endfor %}
+    </div>
   </div>
-</div>
 
 
 {% endblock content %}
 
 {% block footer_script %}
+  <script>
+
+    if (window.location.href.includes("?")) {
+      // We have GET parameters. Populate filter box
+      let qd = {};
+      location.search.substr(1).split("&").forEach(function (item) {
+        (item.split("=")[0] in qd) ? qd[item.split("=")[0]].push(item.split("=")[1]) :
+            qd[item.split("=")[0]] = [item.split("=")[1]]
+      });
+
+      console.log(qd);
+      if ("specialization" in qd) {
+        let checkBoxes = document.getElementsByClassName("specialization-checkbox");
+
+        for (let i = 0; i < checkBoxes.length; i++) {
+          if (qd["specialization"].includes(checkBoxes[i].getAttribute("name"))) {
+            checkBoxes[i].checked = true;
+          } else {
+            checkBoxes[i].checked = false;
+          }
+        }
+      }
+    }
+
```

Comment:
Suggestion: give an even more specific comment for section. For example,
// Enable filtering for interviews using GET parameters (populating filter box)